Sources/ComplexModule/Differentiable.swift

```diff
@@ -15,6 +15,10 @@ import _Differentiation
 extension Complex: Differentiable
 where RealType: Differentiable, RealType.TangentVector == RealType {
   public typealias TangentVector = Self
+
+  public var zeroTangentVectorInitializer: () -> Self {
+    { Complex.zero }
+  }
```

Comment:
I chose not to use a memberwise implementation of `zeroTangentVectorInitializer`:
```swift
  public var zeroTangentVectorInitializer: () -> Self {
    { [realTanInit = real.zeroTangentVectorInitializer,
       imaginaryTanInit = imaginary.zeroTangentVectorInitializer] in
      Complex(realTanInit(), imaginaryTanInit())
    }
  }
```

I don't believe it's necessary for `Complex`.